### ISSUE TITLE
Update and test stage1 and production with the new b/r filtering options

### ISSUE THREAD

**Issue by giohappy** [geosolutions-it/geonode-afghanistan#21]
I'm referring to [this PR](https://github.com/GeoNode/geonode/pull/6053)
Let's do the upgrade, accomodte the filtering in the stage1 settings and run tests.

**Comment by afabiani** [comment]
Done

**Comment by giohappy** [comment]
@afabiani did you run the following tests?

 - create new layer on stage1
 - remove a layer from stage1
 - change permission on one layer on stage1
 - add/remove users from stage1
 - run b/r
 - check the consistency between stage1 and prod, both in GeoNode and Geoserver (no stale data and layers, etc.)

**Comment by afabiani** [comment]
@giohappy yes, even if I did the opposite, backup on production and restore on stage1.

I also tried a new document and unfortunately I noticed that the uploaded folder was not included into the zip.

In this case the uploaded folder is external to geonode, I don't remember exactly which was the logic here. I remember something was done in order to avoid including static and media folders of geonode core, am I correct?

**Comment by afabiani** [comment]
my fault, I wrote a wrong date in the filter

**Comment by giohappy** [comment]
so do you confirm that b/r works as expected for documents?

**Comment by afabiani** [comment]
yes
